torch/csrc/jit/passes/python_print.cpp

```diff
@@ -1043,6 +1043,9 @@ struct PythonPrintImpl {
         }
         stmt << ")";
       } break;
+      case prim::ToList: {
+        stmt << useOf(node->input()) << ".tolist()";
+      } break;
```

Comment:
To be honest, I guessed here, but it looks like the automatic import/export tests don't run, so this isn't being tested at the moment (cc: @eellison)

Reply:
i'm trying to land https://github.com/pytorch/pytorch/pull/33468... hopefully will land soon and you can rebase on top of it.

Reply:
Fixed.